denonavr/foundation.py

```diff
@@ -385,7 +385,8 @@ async def _async_delay_time_callback(
         self, zone: str, event: str, parameter: str
     ) -> None:
         """Handle a delay time change event."""
-        if event != "PS" or parameter[0:3] != "DEL":
+        # do not match "DELAY" as it's another event
+        if event != "PS" or parameter[0:3] != "DEL" or "DELAY" in parameter:
```

Comment:
```suggestion
        if event != "PS" or parameter[0:3] != "DEL" or parameter[0:5] == "DELAY":
```
This would be consistent the matcher in `_async_delay_callback`.

Reply:
Fixed